enterprise/migrations/0001_squashed_0092_add_use_enterprise_catalog_sample.py

```diff
@@ -0,0 +1,1032 @@
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False, verbose_name='created')),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False, verbose_name='modified')),
+                ('entitlement_id', models.PositiveIntegerField(help_text="Enterprise customer's entitlement id for relationship with e-commerce coupon.", unique=True, verbose_name='Seat Entitlement')),
+                ('enterprise_customer', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='enterprise_customer_entitlements', to='enterprise.EnterpriseCustomer')),
+            ],
+            options={
+                'ordering': ['created'],
+                'verbose_name': 'Enterprise Customer Entitlement',
+                'verbose_name_plural': 'Enterprise Customer Entitlements',
+            },
+        ),
+        migrations.CreateModel(
+            name='HistoricalEnterpriseCustomerEntitlement',
+            fields=[
+                ('id', models.IntegerField(auto_created=True, blank=True, db_index=True, verbose_name='ID')),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False, verbose_name='created')),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False, verbose_name='modified')),
+                ('entitlement_id', models.PositiveIntegerField(db_index=True, help_text="Enterprise customer's entitlement id for relationship with e-commerce coupon.", verbose_name='Seat Entitlement')),
+                ('history_id', models.AutoField(primary_key=True, serialize=False)),
+                ('history_date', models.DateTimeField()),
+                ('history_type', models.CharField(choices=[('+', 'Created'), ('~', 'Changed'), ('-', 'Deleted')], max_length=1)),
+                ('enterprise_customer', models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='enterprise.EnterpriseCustomer')),
+                ('history_user', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to=settings.AUTH_USER_MODEL)),
+                ('history_change_reason', models.CharField(max_length=100, null=True)),
+            ],
+            options={
+                'get_latest_by': 'history_date',
+                'ordering': ('-history_date', '-history_id'),
+                'verbose_name': 'historical Enterprise Customer Entitlement',
+            },
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='catalog',
+            field=models.PositiveIntegerField(blank=True, help_text='Course catalog for the Enterprise Customer.', null=True),
+        ),
```

Comment:
There are lots of Alters.  Does it make sense to try to use these to begin with?